<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterationWindow</class>
 <widget class="QWidget" name="RegisterationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberText">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailText">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerationBackButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
